src/NAR/Constructs/DivideConstruct.java

```diff
@@ -117,7 +118,14 @@ public static void main(String args[]) {
 
     @Override
     public String getHelpInformation() {
-        return null;
+        return "Divide:\n"
+                + "   Syntax: Divide <optional \"-\"><variable/number> by <optional \"-\"><variable/number> "
+
+                + "\n   Divides operand 1(left operand) by operand 2(right operand) and prints the result "
```

Comment:
prefer using left/right operand rather than operand 1 and 2. it's easier to understand for the user